src/Learners/Classifier/ClassifierEvaluator.cs

```diff
@@ -316,7 +316,7 @@ public ConfusionMatrix<TLabel> ConfusionMatrix(TInstanceSource instanceSource, I
         /// <param name="instanceSource">The instance source.</param>
         /// <param name="predictions">The predictions.</param>
         /// <returns>The computed precision-recall curve.</returns>
-        public IEnumerable<Pair<double, double>> PrecisionRecallCurve(
+        public IEnumerable<ValueTuple<double, double>> PrecisionRecallCurve(
```

Comment:
This code should not be using "Pair" or "ValueTuple" at all.  It should be using a custom struct with fields "Precision" and "Recall".

Reply:
From the [documentation on Tuples](https://docs.microsoft.com/en-us/dotnet/csharp/tuples): 
Tuples are most useful for utility methods that are private, or internal. Create user-defined types, either class or struct types when your public methods return a value that has multiple elements.